composer.json

```diff
@@ -11,12 +11,18 @@
     "suggest": {
         "dpc-sdp/tide_api:^1.2.6": "Allows to use Drupal in headless mode",
         "dpc-sdp/tide_webform:^1.2.4": "Allows to use questionnaires",
-        "dpc-sdp/tide_news:^1.1.5": "News content type and related configuration for Tide Drupal 8 distribution"
+        "dpc-sdp/tide_news:^1.1.5": "News content type and related configuration for Tide Drupal 8 distribution",
+        "dpc-sdp/tide_automated_listing:dev-feature/TVD-200-Install-card-collection": "Allows to use tide automated listing"
     },
     "repositories": {
         "drupal": {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
+        },
```

Comment:
Is there any reason we're not publishing this to Packagist?

Reply:
The only reason is that it is not ready to be used by sdp :S For VT we will be using this feature branch until sdp is ready and this actually gets merged into tide_landing_page.

Reply:
Ah, makes sense.